app/src/lib/stores/app-store.ts

```diff
@@ -763,8 +772,16 @@ export class AppStore {
     const gitHubRepository = repository.gitHubRepository
     if (gitHubRepository) {
       this._updateIssues(gitHubRepository)
+
+      this.pullRequestStore
+        .getPullRequests(gitHubRepository)
+        .then(p =>
```

Comment:
So we don't wanna `await` this because it'd cause the rest of the method to wait on this to complete, but we don't want that. Alternatively, we could split this into its own method?

Reply:
👍  it's not on the critical path for switching repositories.
👍 it's own method for readability

Reply:
Friendly bump on this, I think it's the only thing left to do.